javascript/challenges/multiBracketValidation/multi-bracket-validation.test.js

```diff
@@ -0,0 +1,11 @@
+'use strict';
+const mvb = require('multi-bracket-validation');
```

Comment:
This line is causing the test not to run, it should be:
`const mvb = require('./multi-bracket-validation');`